tests/foreman/ui/test_org.py

```diff
@@ -524,27 +535,22 @@ def test_add_domain(self, domain_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        domain = entities.Domain(name=domain_name).create()
-        self.assertEqual(domain.name, domain_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_domains=[domain_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_domains'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % domain_name))
-            self.assertIsNotNone(element)
-
-    @data(
-        gen_string('alpha'),
-        gen_string('numeric'),
-        gen_string('alphanumeric'),
-        gen_string('utf8'),
-        gen_string('latin1'),
-    )
-    def test_add_user(self, user_name):
+            for domain_name in generate_strings_list():
```

Comment:
Same comment as before about creating only one organization and then adding/removing domains. Not a blocker for this PR.